etc/debian/control

```diff
@@ -1,8 +1,10 @@
 Source: libmongocrypt
 Priority: optional
-Maintainer: Roberto C. Sanchez <roberto@connexer.com>
-Build-Depends:
-Standards-Version: 4.1.3
+Maintainer: Mongo C Driver Team <mongo-c-driver-debian@googlegroups.com>
+Uploaders: Kevin Albertson <kevin.albertson@mongodb.com>,
+           Samantha Ritter <samantha.ritter@mongodb.com>,
+           Roberto C. Sanchez <roberto@connexer.com>
```

Comment:
Here too, add Clyde so we have all of the C/C++ team listed.

Reply:
Done